packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -396,21 +400,16 @@ export class MLSService extends TypedEventEmitter<Events> {
     });
   }
 
-  public updateKeyingMaterial(conversationId: ConversationId) {
-    return this.processCommitAction(conversationId, () => this.coreCryptoClient.updateKeyingMaterial(conversationId));
+  private updateKeyingMaterial(groupId: string) {
+    const groupIdBytes = Decoder.fromBase64(groupId).asBytes;
```

Comment:
no error handling here if decoding fails? Can we add a try/catch block here?

Reply:
Why would it fail? It would mean that `groupId` was somehow malformed what would be programming error.

Reply:
if groupId is malformed then don't we want to throw some exception?

Reply:
It will be thrown by decrypt method itself, this is not something we should handle